bson/src/main/scala-2.13-/types.scala

```diff
@@ -907,7 +907,7 @@ sealed trait BSONElementSet extends ElementProducer { self: BSONValue =>
   override private[reactivemongo] lazy val byteSize: Int =
```

Comment:
` bson/src/main/scala-2.13+/types.scala` ?

Reply:
I made the change in 2.13+